modules/builtin/src/content-types/audio.js

```diff
@@ -0,0 +1,90 @@
+const base = require('./_base')
+const path = require('path')
+const utils = require('./_utils')
+
+function render(data) {
+  const events = []
+
+  if (data.typing) {
+    events.push({
+      type: 'typing',
+      value: data.typing
+    })
+  }
+
+  return [
+    ...events,
+    {
+      type: 'audio',
+      title: data.title,
+      url: utils.formatURL(data.BOT_URL, data.audio),
+      collectFeedback: data.collectFeedback
+    }
+  ]
+}
+
+function renderElement(data, channel) {
+  // rendering is handled by channel-vonage
+  if (channel === 'vonage') {
+    return data
+  } else {
+    return render(data)
+  }
+}
+
+module.exports = {
+  id: 'builtin_audio',
+  group: 'Built-in Audio',
+  title: 'Audio',
+
+  jsonSchema: {
+    description: 'module.builtin.types.audio.description',
+    type: 'object',
+    $subtype: 'audio',
+    required: ['audio'],
+    properties: {
+      audio: {
+        type: 'string',
+        $subtype: 'audio',
+        $filter: '.mp3',
+        title: 'module.builtin.types.audio.title'
+      },
+      title: {
+        type: 'string',
+        title: 'module.builtin.types.audio.audioLabel'
+      },
+      ...base.typingIndicators
+    }
+  },
+
+  uiSchema: {
+    title: {
+      'ui:field': 'i18n_field'
+    }
+  },
+
+  computePreviewText: formData => {
+    if (!formData.audio) {
+      return
+    }
+
+    const link = utils.formatURL(formData.BOT_URL, formData.audio)
```

Comment:
We don't offer s3 backend to media-service yet but when we do, this won't be the proper URL but I guess it's not important as 
1- we don't have an s3 backend yet
2- this is only for preview

Reply:
I think we could simply render a relative URL. Only channels other than channel-web requires to have an "external" URL. We'll see once we're there!